sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+      label: 'ScalarDB について',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'database-configurations',
+          label: '基本の設定',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-notes',
+          label: 'リリースノート',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'リリースサポートポリシー',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'クイックスタート',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'quick-start-overview',
+          label: '概要',
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Core ライブラリを使用してトランザクションを実行',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb',
+              label: 'ScalarDB Core を使用',
+            },
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb-by-using-kotlin',
+              label: 'Kotlin を使用',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Cluster を使用してトランザクションを実行',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster',
+              label: 'ScalarDB Cluster を使用',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-jdbc',
+              label: 'JDBC 経由で SQL を使用',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-spring-data-jdbc',
+              label: 'Spring Data JDBC 経由で SQL を使用',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-graphql',
+              label: 'GraphQL を使用',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Analytics を使用して分析クエリを実行',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-analytics-postgresql/getting-started',
+              label: 'ScalarDB Analytics with PostgreSQL を使用',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/scalardb-analytics-spark-sample/README',
+              label: 'ScalarDB Analytics を使用',
```

Comment:
This should be at the top of this category since we don't support ScalarDB Analytics with PostgreSQL.
Also, this category is about ScalarDB Analytics, so saying `Use ScalarDB Analytics` sounds redundant.
I'm not sure what else we can say though.
Maybe, `Use Spark` or `Use Spark as an execution engine`. The latter sounds better but maybe too long?